### PR TITLE
Upgraded keepassxc to 2.5.4

### PR DESCRIPTION
**Pull request by vraravam**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked the cask was not [already refused].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by core-code** [comment]
this is marked as a "Pre-release" in github...

**Comment by SteffenGerdes** [comment]
@core-code no longer https://github.com/keepassxreboot/keepassxc/releases/tag/2.5.4

**Comment by core-code** [comment]
ok thanks. can you also update 
`  depends_on macos: '>= :sierra'
`to `high_sierra` 
because this isn't linking to the sierra version (and probably shouldn't)

**Comment by vraravam** [comment]
> ok thanks. can you also update
> ` depends_on macos: '>= :sierra'`to `high_sierra`
> because this isn't linking to the sierra version (and probably shouldn't)

@core-code - thanks for the review - this is also done. Do you want me to squash the commits into one? (My personal pref is to have a single commit - but, your call).

**Comment by core-code** [comment]
thanks for the updates.

i think i can do that on merge, lets try it :)

**Review by vladimyr — COMMENTED** [review]
This should contain both sierra and high sierra download URL & checksum determined by version check.

File: Casks/keepassxc.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
   # github.com/keepassxreboot/keepassxc was verified as official when first introduced to the cask
-  url "https://github.com/keepassxreboot/keepassxc/releases/download/#{version}/KeePassXC-#{version}-2.dmg"
+  url "https://github.com/keepassxreboot/keepassxc/releases/download/#{version}/KeePassXC-#{version}.dmg"
   appcast 'https://github.com/keepassxreboot/keepassxc/releases.atom'
   name 'KeePassXC'
   homepage 'https://keepassxc.org/'
 
-  depends_on macos: '>= :sierra'
+  depends_on macos: '>= :high_sierra'
```
[the remark this answers is not in the document]
Comment (vladimyr) [reply Casks/keepassxc.rb:11]: This was my mistake, official policy is to support the last 2 macOS versions.

**Pull request merged**